test_infra/README.md

```diff
@@ -0,0 +1,19 @@
+# Running tests
```

Comment:
Should this be in apiv2 as it provides more info for that than the test_infra folder?